src/Symfony/Component/Form/Tests/Extension/Validator/ValidatorTypeGuesserTest.php

```diff
@@ -84,6 +87,16 @@ public function testGuessRequired($constraint, $guess)
         $this->assertEquals($guess, $this->guesser->guessRequired(self::TEST_CLASS, self::TEST_PROPERTY));
     }
 
+    /**
+     * @group legacy
+     */
+    public function testLegacyGuessRequired($constraint, $guess)
+    {
+      if (PHP_VERSION_ID < 70000) {
+        $this->testGuessRequired(new True(), new ValueGuess(true, Guess::HIGH_CONFIDENCE));
```

Comment:
Also would be better to move the test outside of this if and call markTestSkipped here